wallet/src/main/scala/org/constellation/wallet/BIP44.scala

```diff
@@ -0,0 +1,58 @@
+package org.constellation.wallet
+
+import java.math.BigInteger
+import java.security.{KeyFactory, KeyPair, PrivateKey, PublicKey}
+import java.util
+import java.util.Date
+
+import org.bitcoinj.crypto.{ChildNumber, DeterministicKey, HDUtils}
+import org.bitcoinj.wallet.{DeterministicKeyChain, DeterministicSeed}
+import org.bouncycastle.asn1.sec.SECNamedCurves
+import org.bouncycastle.asn1.x9.X9ECParameters
+import org.bouncycastle.jce.spec.{ECParameterSpec, ECPrivateKeySpec, _}
+import org.bouncycastle.math.ec
+import org.constellation.keytool.KeyUtils
+import org.constellation.keytool.KeyUtils.insertProvider
+import org.constellation.wallet.BIP44.{getPrivateKeyFromECBigIntAndCurve, getPublicKeyFromECPoint}
+
+class BIP44(seedPrase: String, childIndex: Int = 0, passphrase: String = "", creationTime: Long = new Date().getTime) {
+  val chainPathPrefix: String = "M/44H/137036H/0H/0/" //137.036 is the fine structure constant ...
```

Comment:
Isn't the coin_type level supposed to be and ID for DAG from SLIP-44? So:
1137 | 0x80000471

Reply:
Must've forgotten that I already registered our coin type, yep it should be 1137

Reply:
I pushed one commit changing coin type value to 1137